bucket/furmark.json

```diff
@@ -0,0 +1,27 @@
+{
+    "homepage": "https://geeks3d.com/furmark/",
+    "description": "A GPU Stress Test and OpenGL Benchmark.",
+    "license": "Freeware",
+    "version": "1.20.8.0",
+    "url": "https://www.geeks3d.com/downloads/2019/FurMark_1.20.8.0_Setup.exe",
+    "hash": "0b212989d8a3aa01da79ff45f03701aa0bb7ba08f2b0329d235320ff2635afb8",
+    "innosetup": true,
+    "bin": "FurMark.exe",
+    "shortcuts": [
+        [
+            "FurMark.exe",
+            "FurMark"
+        ]
+    ],
+    "checkver": {
+        "url": "https://geeks3d.com/furmark/downloads/",
+        "regex": "<b>Version (?<version>[\\d.]+) - (?<year>\\d+)-\\d+-\\d+</b>"
+    },
+    "autoupdate": {
+        "url": "https://www.geeks3d.com/downloads/$matchYear/FurMark_$version_Setup.exe",
+        "hash": {
+            "url": "https://geeks3d.com/furmark/downloads/",
+            "regex": "$basename[\\S\\s]+SHA256\\s+=>\\s+$sha256\\s+<!--"
```

Comment:
- In general `.*?` is prefered in other manfiests over `[\\S\\s]+`
- Is it needed to be so strict?

```suggestion
            "regex": "$basename.*?SHA256\\s+=>\\s+$sha256\\s+<!--"
```

```suggestion
            "regex": "$basename.*?+SHA256\\s+=>\\s+$sha256"
```

Reply:
I can't remember it, even if it's so simple 😁

Reply:
You could pretty much remove the comment too. Comment could be easily changed / removed.

Reply:
By using `[\S\s]+` it would skip the correct checksum and select the last one.

Reply:
Hm, `.*?` doesn't work.
https://regex101.com/r/g30jIy/1

Reply:
You need multiline `(?sm)$basename.*?SHA256\\s+=>\\s+$sha256`